MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressDxe.inf

```diff
@@ -75,6 +76,7 @@
   gEfiStorageSecurityCommandProtocolGuid      ## BY_START
   gEfiDriverSupportedEfiVersionProtocolGuid   ## PRODUCES
   gEfiResetNotificationProtocolGuid           ## CONSUMES
+  gEfiMediaSanitizeProtocolGuid               ## CONSUMES

```

Comment:
isn't this produces?

Reply:
Not sure actually. I view this protocol similar to Block Io, Disk Info, Storage Security, etc.

If "produces" is correct, I can change.

Reply:
Understood. Will make "produces".